dace/config_schema.yml

```diff
@@ -770,6 +770,184 @@ required:
+                    decomposition_params:
+                        type: list
+                        title: Numerical values of input parameters (list of tuples)
+                        description: >
+                            Should SOAP generate distribute data decomposition, numerical values
+                            of input parameters are required: number of processors P, size of fast memory Ss,
+                            and dimension sizes. E.g.,: [("p", 8), ("Ss", 32*1024), ("S0", 512), ("S1", 512), ("S2", 512)]
+
+            solver:
+                type: dict
+                title: Solver
+                description: Symbolic solver parameters
+                required:
+                    db_path:
+                        type: string
+                        title: Solver's database path
+                        default: ""
+                        description: >
+                            Path to the json file that holds already solved problems that
+                            can be reused if the Matlab solver is not available. 
+
+                    local_solver_path:
+                        type: string
+                        title: Local matlab solver
+                        default: ""
+                        description: >
+                            Path to local Matlab solver's launch file.
+
+                    remote_solver_address:
+                        type: string
+                        title: Remote matlab solver
+                        default: ""
+                        description: >
+                            Address of the remote Matlab server.
+
+                    only_db:
+                        type: bool
+                        title: Turns off Matlab solver. Relying only on precomputed solutions
+                        default: true
+                        description: >
+                            If only_db is specified, only the problems already analyzed in the
+                            solver's database can be processed. If new problems are encountered,
+                            the solver throws an error.
+
+                    remote_matlab:
+                        type: bool
+                        title: Using remote matlab solver
+                        default: true
+                        description: >
+                            If not relying on the solver's database (solver_only_database),
+                            it specifies whether one uses the local or remote Matlab server
+
+                    caching_solver_solutions:
+                        type: bool
+                        title: Caching newly computed solver results
+                        default: true
+                        description: >
+                            If matlab solver is used and problems that are not in the solver's database 
+                            are encountered, it updates the database with the new solutions.
+
+
+            # output generation
```

Comment:
Do these need to be configuration parameters? You could have an extra method that takes the output SDG statements and formats them accordingly.